apps/chat/src/components/Marketplace/ApplicationCard.tsx

```diff
@@ -137,6 +142,7 @@ export const ApplicationCard = ({
         display:
           !isMyEntity &&
           selectedTab === MarketplaceTabs.MY_APPLICATIONS &&
+          installedModelIds.has(entity.reference) &&
```

Comment:
remove previous condition `selectedTab === MarketplaceTabs.MY_APPLICATIONS &&`

Reply:
If we remove condition , the remove button will be shown in DIAL Marketplace , and it shouldn't be shown there

Reply:
I suppose we agreed that all actions will be available on both marketplace and workspace